generators/app/index.js

```diff
@@ -77,13 +78,17 @@ module.exports = class extends Generator {
                         value: 'ext-command-js'
                     },
                     {
-                        name: 'New Dashboard Insight',
-                        value: 'ext-insight'
```

Comment:
are we removing dashboard insight and replacing with new dashboard template?

Reply:
Yes, I think so, since dashboard template does more than just insight